src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,94 @@
+.menu {
+  padding: 48px 240px;
+
+  @include for-tablet {
+    padding: 48px 100px;
+  }
+
+  @include for-phone {
+    padding: 24px 20px;
+  }
+
+  &__nav {
+    margin-bottom: 50px;
+  }
+
+  &__list {
+    list-style: none;
```

Comment:
I think you could simply write in your reset file code to reset default list styles, as well as link styles (text-decoration) so you don`t have to write it every single time to reset the style.

Reply:
Fixed